web/src/components/dashboards/index.tsx

```diff
@@ -905,6 +906,9 @@ const MonitoringDashboardsPage_: React.FC<MonitoringDashboardsPageProps> = ({ hi
   const [board, setBoard] = React.useState<string>();
   const [boards, isLoading, error] = useFetchDashboards(namespace);
 
+  const { getDashboards } = usePersesClient();
+  const [persesBoards] = getDashboards;
```

Comment:
This probably should be a function so it can be lazy executed.
```suggestion
  const persesBoards = getDashboards();
```

Reply:
Removed in the latest push.